Tests/Realm.Tests/Sync/DataTypeSynchronizationTests.cs

```diff
@@ -22,16 +22,26 @@
 using System.Linq;
 using System.Reflection;
 using System.Threading.Tasks;
+using Baas;
 using MongoDB.Bson;
+using Nito.AsyncEx;
 using NUnit.Framework;
+using NUnit.Framework.Constraints;
+using Realms.Exceptions;
 using Realms.Extensions;
 using Realms.Helpers;
+using Realms.Logging;
+using Realms.Sync;
+using Realms.Sync.Exceptions;
+using Realms.Tests.Database;
```

Comment:
I don't think all of these are used, are they?

Reply:
The comparer is from `Realms.Tests.Database` but there were a lot of unused `using`. I have cleaned it up.